ethereumj-core/src/main/java/org/ethereum/facade/EthereumImpl.java

```diff
@@ -101,14 +101,11 @@ public PeerInfo findOnlinePeer(Set<PeerInfo> excludePeers) {
         worldManager.startPeerDiscovery();
 
         final Set<PeerInfo> peers = worldManager.getPeerDiscovery().getPeers();
-        synchronized (peers) {
```

Comment:
because it made no sense before - was syncronized to a local variable - so just removed a nop